.github/workflows/test.yml

```
@@ -136,9 +135,19 @@ jobs:
         uses: actions/setup-python@v4
         with:
           python-version: "${{ matrix.python }}"
+
       - name: Install Python dependencies
         run: |
           pip install --upgrade pip
+
+          if [ "${{ matrix.python }}" == "3.11"* ]; then
```

```suggestion
          if [[ "${{ matrix.python }}" == "3.11"* ]]; then
```

I think this is required.